composer.json

```diff
@@ -28,5 +28,17 @@
         "psr-4": {
             "Facebook\\InstantArticles\\": "tests/Facebook/InstantArticles/"
         }
+    },
+    "scripts": {
+        "all": [
+            "composer cs",
```

Comment:
We can simply use `@cs` here (see https://getcomposer.org/doc/articles/scripts.md#referencing-scripts)